rules/Transform/Rector/ArrayDimFetch/ArrayDimFetchToMethodCallRector.php

```diff
@@ -31,41 +39,54 @@ public function getRuleDefinition(): RuleDefinition
         return new RuleDefinition('Change array dim fetch to method call', [
             new ConfiguredCodeSample(
                 <<<'CODE_SAMPLE'
-$app['someService'];
+$object['key'];
+$object['key'] = 'value';
+isset($object['key']);
+unset($object['key']);
 CODE_SAMPLE
                 ,
                 <<<'CODE_SAMPLE'
-$app->make('someService');
+$object->get('key');
+$object->set('key', 'value');
+$object->has('key');
+$object->unset('key');
 CODE_SAMPLE
                 ,
-                [new ArrayDimFetchToMethodCall(new ObjectType('SomeClass'), 'make')]
+                [new ArrayDimFetchToMethodCall(new ObjectType('SomeClass'), 'get', 'set', 'has', 'unset')],
             ),
         ]);
     }
 
     public function getNodeTypes(): array
     {
-        return [ArrayDimFetch::class];
+        return [ArrayDimFetch::class, Assign::class, Isset_::class, Unset_::class];
```

Comment:
move isset/unset/assign as wrapper in the left of ArrayDimFetch so skip deep child can clearly early checked

Reply:
```suggestion
        return [Assign::class, Isset_::class, Unset_::class, ArrayDimFetch::class];
```